This drops a test that fails with numpy 1.12.

Upstream bug URL: https://github.com/statsmodels/statsmodels/issues/3541

--- a/statsmodels/discrete/tests/test_discrete.py     2017-03-06 15:29:19.947343331 +0000
+++ b/statsmodels/discrete/tests/test_discrete.py     2017-03-06 15:29:37.759328845 +0000
@@ -1329,21 +1329,6 @@
         res = mod.fit(start_params=-np.ones(4), method='newton', disp=0)
     assert_(not res.mle_retvals['converged'])
 
-def test_issue_339():
-    # make sure MNLogit summary works for J != K.
-    data = sm.datasets.anes96.load()
-    exog = data.exog
-    # leave out last exog column
-    exog = exog[:,:-1]
-    exog = sm.add_constant(exog, prepend=True)
-    res1 = sm.MNLogit(data.endog, exog).fit(method="newton", disp=0)
-    # strip the header from the test
-    smry = "\n".join(res1.summary().as_text().split('\n')[9:])
-    cur_dir = os.path.dirname(os.path.abspath(__file__))
-    test_case_file = os.path.join(cur_dir, 'results', 'mn_logit_summary.txt')
-    test_case = open(test_case_file, 'r').read()
-    np.testing.assert_equal(smry, test_case[:-1])
-
 def test_issue_341():
     data = sm.datasets.anes96.load()
     exog = data.exog
